src/Assets/Standard Assets/Bugsnag/BugsnagBehaviour.cs

```diff
@@ -50,10 +50,12 @@ void Awake()
     /// <param name="hasFocus"></param>
     void OnApplicationFocus(bool hasFocus)
     {
-      if (hasFocus && Bugsnag.Client.Configuration.AutoCaptureSessions)
-      {
-        Bugsnag.Client.SessionTracking.StartSession();
-      }
+      Bugsnag.Client.SetApplicationState(hasFocus);
+    }
+
+    void OnApplicationPause(bool paused)
+    {
+      Bugsnag.Client.SetApplicationState(!paused);
```

Comment:
This is very minor but the variable naming/method signature here reads as though the application is being set as not paused (even though the opposite is the case)